packages/iris-grid/src/sidebar/AdvancedSettingsMenu.tsx

```diff
@@ -34,9 +34,9 @@ AdvancedSettingsMenu.getAdvancedSettingsTitleForKey = (
 ): string => {
   switch (key) {
     case AdvancedSettingsType.FILTER_CONTROL_CHANGE_CLEARS_ALL_FILTERS:
-      return 'Control filters clear existing table filters';
+      return 'Clear current table filters before applying new filters from a control';
     case AdvancedSettingsType.LINK_CHANGE_CLEARS_ALL_FILTERS:
-      return 'Inbound links clear existing table filters';
+      return 'Clear current table filters before applying new filters from a incoming link filter';
```

Comment:
I'm not crazy about these wordings, they show up in the menu pretty long.

![image](https://user-images.githubusercontent.com/4505624/221982160-6ea13dea-cc80-46bb-94a0-70019050dcf1.png)

I think I somewhat prefer the older text, but up to you.

Reply:
I think for advanced settings is probably fine to be more clear because it's a weird option.